cunumeric/module.py

```diff
@@ -1314,33 +1361,45 @@ def _concatenate(
     dtype=None,
     casting="same_kind",
     common_info=None,
+    slicing_only=False,
 ):
     if axis < 0:
         axis += len(common_info.shape)
     leading_dim = _builtin_sum(arr.shape[axis] for arr in inputs)
     out_shape = list(common_info.shape)
```

Comment:
Again, `common_info` seems to be optional, but you always take the shape of it. If it is never a `None`, then please turn it into a positional argument to document the behavior.